cmd/telemeter-server/main.go

```diff
@@ -403,17 +405,12 @@ func (o *Options) Run() error {
 			}
 
 			auth := jwt.NewAuthorizeClusterHandler(o.Logger, o.PartitionKey, o.TokenExpireSeconds, signer, o.RequiredLabels, clusterAuth)
-			validator := validate.New(o.PartitionKey, o.LimitBytes, 24*time.Hour, time.Now)
+			//validator := validate.New(o.PartitionKey, o.LimitBytes, 24*time.Hour, time.Now)
```

Comment:
are we intentionally ignoring this? do we just want to delete the line?

Reply:
commented out during refactoring for reference. The `validate.New` func doesn't even exist anymore. Will remove.